common/text/token_info_test_util.h

```diff
@@ -34,7 +34,10 @@ namespace verible {
 struct ExpectedTokenInfo : public TokenInfo {
   // This pseudo token_enum signals to the test harness to not bother
   // checking the token enum, and check only the string contents.
-  enum { kDontCare = -1 };
+  enum {
+    kDontCare = -2,
+    kSkip
```

Comment:
Explain the difference between these two.

Reply:
`kDontCare` compares length of the token. `kSkip` doesn't compare anything, just skip token.

Reply:
Let's rename this `kNoToken` and describe in comment here.